vertx-web/src/test/java/io/vertx/ext/web/handler/StaticHandlerTest.java

```diff
@@ -252,6 +252,89 @@ public void testHttp2Push() throws Exception {
     latch.await();
   }
 
+  @Test
+  public void testSkipCompressionForMediaTypes() throws Exception {
+    Set<String> compressedMediaTypes = new HashSet<>();
+    compressedMediaTypes.add("image/jpeg");
+    stat.skipCompressionForMediaTypes(compressedMediaTypes);
+    router.route().handler(stat);
+
+    waitFor(4);
+    HttpServer httpServer = vertx.createHttpServer(new HttpServerOptions().setCompressionSupported(true));
+    httpServer.requestHandler(router).listen(8444);
+    HttpClient client = vertx.createHttpClient();
```

Comment:
`WebTestBase` creates a `client` instance that you can use instead of creating a new one.